.changeset/feat-TreeView-add-expandAll-and-collapseAll.md

```diff
@@ -0,0 +1,5 @@
+---
+'react-magma-dom': minor
+---
+
+feat(TreeView): add `expandAll` and `collapseAll`.
```

Comment:
Maybe we also mention here that we remove collapse/expand animation in case our adopters notice